packages/rescript-relay-router/src/RelayRouter.res

```diff
@@ -48,10 +48,7 @@ module RouterEnvironment = {
 }
 
 module Router = {
-  let dictDelete: (
-    Js.Dict.t<'any>,
-    string,
-  ) => unit = %raw(`function(dict, key) { delete dict[key] }`)
+  let dictDelete: (Dict.t<'any>, string) => unit = %raw(`function(dict, key) { delete dict[key] }`)
```

Comment:
Can remove this and use `Dict.delete` directly from Core, right?

Reply:
sure, I didn't see that, it just got reformatted, I'll remove it.